builders/get-started/eth-compare/tx-fees.md

```diff
@@ -0,0 +1,190 @@
+
+## Substrate API Transaction Fees {: #substrate-api-transaction-fees }
+
+The fees of a transaction sent via the Substrate API on Moonbeam can be read directly from a Sidecar block JSON object. The nesting structure is as follows:
+
+```JSON
+RESPONSE JSON Block Object:
+    |--extrinsics
+        |--{extrinsic number}
+            |--method
+            |--signature
+            |--nonce
+            |--args
+            |--tip           
+            |--hash
+            |--info
+            |--era
+            |--events
+                |--{event number}
+                    |--method
+                        |--pallet: "transactionPayment"
+                        |--method: "TransactionFeePaid"
+                    |--data
+                        |--0
+                        |--1
+                        |--2
+    ...
+
+```
+
+The object mappings are summarized as follows:
+
+|     Tx Information      |                           Block JSON Field                            |
+|:-----------------------:|:---------------------------------------------------------------------:|
+| Fee paying account | `extrinsics.{extrinsic number}.events.{event number}.data.0`  |
+|  Total fees paid  | `extrinsics.{extrinsic number}.events.{event number}.data.1` |
+|     Tip      | `extrinsics.{extrinsic number}.events.{event number}.data.2` |
+
+The transaction fee related information can be retrieved under the event of the relevant extrinsic where the `method` field is set to: 
+
+```
+pallet: "transactionPayment", method: "TransactionFeePaid" 
+```
+
+And then the total transaction fee paid for this extrinsic is mapped to the following field of the block JSON object:
+
+```
+extrinsics.{extrinsic number}.events.{event number}.data.1
+```
+
+## Ethereum API Transaction Fees {: #ethereum-api-transaction-fees }
+
+### Calculating Ethereum API Transaction Fees {: #calculating-ethereum-api-transaction-fees }
+
+To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum API, the following formula can be used:
+
+=== "EIP-1559"
+    ```
+    Transaction Fee =（Base Fee + Max Priority Fee Per Gas) * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    ```
+=== "Legacy"
+    ```
+    Transaction Fee = Gas Price * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    ```
+=== "EIP-2930"
+    ```
+    Transaction Fee = Gas Price * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    ```
+
+The values of `Gas Price` and `Max Priority Fee Per Gas` for the applicable transaction types can be read from the block JSON object according to the data structure described in [the Sidecar API page](/builders/build/substrate-api/sidecar/#evm-fields-mapping-in-block-json-object){target=_blank}. 
```

Comment:
would it be too much to add the values here? just so the user doesn't have to flip back and forth between the two pages

Reply:
Added the relevant fields.